stock_picking_partner_language/README.rst

```diff
@@ -0,0 +1,41 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
There's missing the target line (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)